src/interfaces/ecpg/pgtypeslib/dt_common.c

```diff
@@ -1864,8 +1866,6 @@ DecodeDateTime(char **field, int *ftype, int nf,
 	int			haveTextMonth = FALSE;
 	int			is2digits = FALSE;
 	int			bc = FALSE;
-	int			t = 0;
-	int		   *tzp = &t;
```

Comment:
Why did these variables exist in the first place? It seems like they aren't used for anything even before the patch.